charts/policy-controller/README.md

```diff
@@ -1,8 +1,8 @@
 # policy-controller
 
-![Version: 0.5.8](https://img.shields.io/badge/Version-0.5.8-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: 0.7.0](https://img.shields.io/badge/AppVersion-0.7.0-informational?style=flat-square)
+![Version: 0.6.4](https://img.shields.io/badge/Version-0.5.8-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: 0.8.2](https://img.shields.io/badge/AppVersion-0.7.0-informational?style=flat-square)
```

Comment:
nit, do you need to update the second AppVersion to 0.8.2?

Reply:
Chart.yaml already has 0.8.2. I just updated the doc :)

Reply:
I mean the link, should the link be changed to:
`https://img.shields.io/badge/AppVersion-0.8.2-informational?style=flat-square`

Reply:
Sorry. Yes. I updated it :)